config/locales/en/sdg.yml

```
@@ -458,7 +461,9 @@ en:
       goal_list: "You can choose one or several SDGs aligned with your %{record}"
       help:
         title: "Which SDGs and targets are aligned with my %{record}?"
-      hint: "You can introduce the code of a specific goal/target or a text to find one"
+        text: "help"
+      hint: You can introduce the code of a specific goal/target or a text to find one. For more information visit the %{link} page.
+      label: You can introduce the code of a specific goal/target or a text to find one
```

With the current text, a screen reader would say: "You can introduce the code of a specific goal/target or a text to find one, Edit, You can introduce the code of a specific goal/target or a text to find one. For more information visit the help page.", which IMHO is a bit redundant.

```suggestion
      label: Targets
```

Not sure using just "Targets" is right, though, :thinking:, since this field allows selecting goals as well. Any suggestions?